changelogs/fragments/lldp-use-get_bin_path-to-locate-the-lldpctl-executable.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

If you think this as a bugfix PR, you should change this to
```suggestion
bugfixes:
```